src/components/Tabs/components/List/tests/List.test.tsx

```diff
@@ -85,18 +88,17 @@ describe('<List />', () => {
     ];
 
     it('renders an Item for each item in disclosureTabs', () => {
-      const list = mountWithAppProvider(
+      const list = mountWithApp(
         <List {...mockProps} disclosureTabs={disclosureTabs} />,
       );
-      expect(list.find(Item)).toHaveLength(2);
+      expect(list).toContainReactComponentTimes(Item, 2);
     });
 
     it('renders the provided content', () => {
-      const list = mountWithAppProvider(
+      const list = mountWithApp(
         <List {...mockProps} disclosureTabs={disclosureTabs} />,
       );
-      const firstItem = list.find(Item).first();
-      expect(firstItem.contains('Repeat customers')).toBe(true);
+      expect(list).toContainReactText('Repeat customers');
```

Comment:
Would 
```expect(list.find(Item)).toContainReactText('Repeat customers');```
Better translate from the old test?

Reply:
Makes sense! I will go for that option! Thanks for the suggestion!